CHANGELOG_UNRELEASED.md

```diff
@@ -13,6 +13,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - in `ssrint.v`
   + lemmas `intrN`, `intrB`
 
+- in `ssrnum.v`
+  + lemma `invf_plt`
+
 ### Changed
```

Comment:
Could you also add an entry for the generalization of invf_lt1 (if I understand the title of the PR correctly)?

Reply:
It cannot replace transparently invf_lt1 so maybe that is not a generalization the way we have been understanding it in the changelog.